src/main.ts

```diff
@@ -15,6 +15,9 @@ async function run() {
     await exec.exec("sudo", ["mkdir", "-p", "/etc/nix"]);
     await exec.exec("sudo", ["echo", "http2 = false", ">>", "/etc/nix/nix.conf"]);
 
+    // Set jobs to number of cores
+    await exec.exec("sudo", ["echo", "max-jobs = auto", ">>", "/etc/nix/nix.conf"]);
```

Comment:
What magic is this?

```
$ sudo echo foo '>>' /root/foo
[sudo] password for user: 
foo >> /root/foo

$ sudo cat /root/foo
cat: /root/foo: No such file or directory

```

Reply:
Oh it actually just doesn't work.

```suggestion
    await exec.exec("sudo", ["sh", "-c", "echo 'max-jobs = auto' >>/etc/nix/nix.conf"]);
```

Reply:
Wrong kind, next try :)